WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreSetupProgressView.swift

```diff
@@ -7,6 +7,10 @@ struct StoreSetupProgressView: View {
 
     let numberOfTasksCompleted: Int
 
+    let shareFeedbackAction: (() -> Void)?
```

Comment:
nit: any reasons why this action handling isn't required instead of being optional?

Reply:
Good question! I had to keep it as an optional because the feedback option won't be displayed when the `StoreSetupProgressView` is presented in expanded mode. i.e. The ellipsis button is only visible in collapsed mode.

Reply:
that's true, makes sense for this to be optional then 👍